GoldenFox/ScheduleParser.cs

```diff
@@ -18,13 +30,15 @@ public class ScheduleParser
         public ScheduleParser()
         {
             _tokenizer = new Tokenizer();
-            Add("every", "day", "month", "week", "year", "hour", "minute",
+            Add("every", "day", "month", "week", "year", "hour", 
+                "minute", "minutes", "m,", "second", "seconds",
                 "at", "@", ":",
                 "st", "nd", "rd", "th",
                 "last",
-                "s",
+                "s", "m",
                 "and",
-                "between");
+                "between",
+                "hh:mm:", "hh:", "mm:");
```

Comment:
I don't think so if the new at/@/between syntax ( "hh:xx"/"hh:xx:yy", "hh:mm:xx"/"mm:xx" & "hh:mm(:ss)" ) in the diagram is to work.